### ISSUE TITLE
PHP Fatal error

### ISSUE THREAD

**Issue by GarfieldsPipe** [Anonyymi/miniboard#12]
`[error] 22078#22078: *1 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Slim\Exception\HttpNotFoundException: Not found. in /var/www/html/vendor/slim/slim/Slim/Middleware/RoutingMiddleware.php:91`

**Comment by Anonyymi** [comment]
Could you please provide some context that lead to this exception? Did you install dependencies with php composer?

**Comment by GarfieldsPipe** [comment]
> Did you install dependencies with php composer?

Yes, I did. Using PHP 8.1.

**Comment by Anonyymi** [comment]
I cannot replicate the issue. Works fine on my Gentoo linux machine with PHP 8.1

**Comment by denamuumi** [comment]
> `[error] 22078#22078: *1 FastCGI sent in stderr: "PHP message: PHP Fatal error: Uncaught Slim\Exception\HttpNotFoundException: Not found. in /var/www/html/vendor/slim/slim/Slim/Middleware/RoutingMiddleware.php:91`
Is the problem persisting after #14 ?

**Comment by Anonyymi** [comment]
Closing because no response and this is an irrelevant issue. Create a new issue if you still have problems.
